integration/js/README.md

```diff
@@ -2,13 +2,96 @@
 
 This guide will help you setup and run integration tests for [Chime SDK](https://github.com/aws/amazon-chime-sdk-js) using [Chime SDK Browser Demo](https://github.com/aws/amazon-chime-sdk-js/tree/master/demos/browser).
 
+## Testing locally
+
+To run integration tests you will need:
+
+- Node 10 or higher. Node 14 is supported.
+- npm 6.11 or higher. 6.14.8 is supported.
+- [KITE](https://github.com/webrtc/KITE).
+
+### Installing KITE on macOS
+
+Run the following.
+
+```shell
+# Install Maven and browser drivers.
+cd integration/js
+brew install maven geckodriver chromedriver
+
+# Wherever you like.
+export KITEDIR=~/repos
+cd $KITEDIR
+git clone https://github.com/webrtc/KITE.git
+cd KITE
+
+# We need a specific commit.
+git checkout e9296165cd24bea92cddb59f2bf211c99f764d58
+chmod +x configureMac.sh
+./configureMac.sh
+```
+
+Choose 'y'.
+
+Now find the Terminal window that opened and complete interactive setup. A good configuration is:
+
+```
+INSTALL_BROWSERS=FALSE
+
+Browser versions
+CHROME_VERSION=86
+FIREFOX_VERSION=84
+
+Whether to use localhost or the computers IP as the grids hub address
+LOCALHOST=TRUE
+
+GeckoDriver and ChromeDriver versions
+GECKO_VERSION=0.27.0
+CHROMEDRIVER_VERSION=86.0.4240.22
+
+Selenium Standalone Server version
+SELENIUM_VERSION_SHORT=3.141
+SELENIUM_VERSION=3.141.59
+```
+
+Finish KITE setup:
+
+```shell
+cd ~/repos/KITE
+export KITE_HOME="$PWD"
+export KITE_JAR="$KITE_HOME/KITE-Engine/target"
+export PATH="$PATH:$KITE_HOME/scripts/mac/path/"
+c
+```
+
+### Installing dependencies
+
+```shell
+cd integration/js
+npm install
+cd -
+```
+
+### Running the tests
+
+```shell
+export KITE_JAR="$KITE_HOME/KITE-Engine/target"
+cd integration
+export SELENIUM_GRID_PROVIDER=local
+```
+
+```shell
+$KITE_HOME/scripts/mac/path/r configs/meeting_end_test.config.json
```

Comment:
Could we change meeting_end_test.config.json to something like <test_name>.config.json or say for example to run meeting end test:

Reply:
We already add $KITE_HOME/scripts/mac/path/ to $PATH above so we could just do r configs/.. right?

Reply:
No — `r` is a shell builtin, so even if it's in your path, it won't work.